core/src/main/java/com/expediagroup/streamplatform/streamregistry/core/extension/validation/ClusterValidator.java

```diff
@@ -0,0 +1,55 @@
+/**
```

Comment:
Why does this show up as new when it's actually been moved right? I know it can be a bit fiddly deleting and moving stuff around but would be nice if they could keep their history...

Reply:
It's just git. Nothing I can do about AFAIK

Reply:
I think you can do "git mv" commands but usually the IDE handles it for you, seems like something went wrong here and with some other files in this PR, oh well.

Reply:
TIL